runtime/gc_modron_startup/mminit.cpp

```diff
@@ -2259,6 +2259,15 @@ combinationMemoryParameterVerification(J9JavaVM *javaVM, IDATA* memoryParameters
 			subSpaceTooLargeOption = "-Xmns";
 			goto _subSpaceTooLarge;
 		}
```

Comment:
there is this set of curly brackets. either the new check should be inside, or this set of brackets should be removed

Reply:
ah, the new code is indeed inside big block of Balanced GC code, by indentation of this closing curly bracket is not correct (which misled me to believe the big block is closed)